magellan-library/src/main/java/com/wealthfront/magellan/init/Magellan.kt

```diff
@@ -0,0 +1,33 @@
+package com.wealthfront.magellan.init
+
+import androidx.annotation.RestrictTo
+import androidx.annotation.RestrictTo.Scope.LIBRARY_GROUP_PREFIX
+import com.wealthfront.magellan.init.Magellan.customDefaultTransition
+import com.wealthfront.magellan.init.Magellan.disableAnimations
+import com.wealthfront.magellan.init.Magellan.logDebugInfo
+import com.wealthfront.magellan.transitions.DefaultTransition
+import com.wealthfront.magellan.transitions.MagellanTransition
+
+public object Magellan {
+
+  internal var logDebugInfo: Boolean = false
+  internal var disableAnimations: Boolean = false
+  internal var customDefaultTransition: MagellanTransition = DefaultTransition()
+
+  public fun init(
+    disableAnimations: Boolean = false,
+    defaultTransition: MagellanTransition = DefaultTransition(),
+    logDebugInfo: Boolean = false,
+  ) {
+    this.logDebugInfo = logDebugInfo
+    this.disableAnimations = disableAnimations
+    this.customDefaultTransition = defaultTransition
+  }
+}
+
+internal fun shouldRunAnimations(): Boolean = !disableAnimations
+
+internal fun shouldLogDebugInfo(): Boolean = logDebugInfo
+
+@RestrictTo(LIBRARY_GROUP_PREFIX)
+public fun getDefaultTransition(): MagellanTransition = customDefaultTransition
```

Comment:
This is used in magellan-legacy so needs to be public but it can only be used inside our libraries